source/reference/limitations.txt

```diff
@@ -101,9 +101,35 @@ Multiple Clusters
 - Syncing one source cluster to many destination clusters is not
   supported.
 
+System Collections
+------------------
+
+{+c2c-product-name+} does not replicate :ref:`system collections 
+<metadata-system-collections>` to the destination cluster.
+
+If the user issues a :dbcommand:`dropDatabase` command on the source cluster,
```

Comment:
[suggestion]

Let's use second person here.

https://www.mongodb.com/docs/meta/style-guide/writing/write-to-the-user/

Reply:
Fixed